sphinxcontrib/aafig.py

```diff
@@ -100,7 +100,8 @@ def render_aafig_images(app, doctree):
     format_map = app.builder.config.aafig_format
     merge_dict(format_map, DEFAULT_FORMATS)
     if aafigure is None:
-        app.builder.warn('aafigure module not installed, ASCII art images '
+         if app.builder.format != 'man':
+            app.builder.warn('aafigure module not installed, ASCII art images '
```

Comment:
Could you emit a [warning.warn()](https://docs.python.org/3/library/warnings.html) if `app.builder.format == 'man'`? Otherwise users will be left clueless about why things are not working.

This applies to this line and all other lines with a `if app.builder.format != 'man':`.

Reply:
Sure! Thanks a lot for that detailed code review! Totally appreciate that and I feel a bit ashamed now that I didn't put more effort in the first place to provide code with higher quality!

Reply:
No need to be ashamed :)
I can totally understand that people, under the uncertainty of how a project's maintainer could take a PR or if it will reply to it at all, would lean towards the *lazy* side :)

And again, thanks a lot for going through the effort to contribute this and even more for following up on it! :pray: